DocumentedExamples/Age_at_the_Bottom.ipynb

```diff
@@ -12,6 +12,8 @@
     "\n",
```

Comment:
<p>This cell was not evaluated properly, so the time is no longer showing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/224/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>